### PR TITLE
[PURIS-DCM] Supply Chain Disruption Notifications KIT 24.08

### PR DESCRIPTION
**Pull request by dominikoeh**
### Added

- Initial version of the Kit including adoption, development and operations view

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by dominikoeh** [comment]
cc @jSchuetz88

**Comment by arnoweiss** [comment]
How were the figures generated? Is it possible to provide editable files (ideally puml or mermaid)?

File: docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+}
+```
+
+It is recommended to use the following values for rightOperand of FrameworkAgreement and UsagePurpose depending on the
+use case:
+
+| Use case | cx-policy:FrameworkAgreement | cx-policy:UsagePurpose |
+| -------- | ---------------------------- | ---------------------- |
+| DCM      | `DemandCapacity:1.0`         | `cx.dcm.base:1`        |
+| PURIS    | `Puris:1.0`                  | `cx.puris.base:1`      |
```
Comment (jSchuetz88) [inline docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md:132]: Did someone checked this part? In my understanding the FrameworkAgreement "Puris:1.0" as well as "DemandCapacity:1.0" will be deprecated with the release of Jupiter.

see: https://github.com/catenax-eV/cx-odrl-profile/blob/main/profile.md

> Puris:1.0
> Valid from: 2024-06-20
> Valid until: 2024-10-16
> Status: deprecated

**Review by jSchuetz88 — CHANGES_REQUESTED** [review]
Hi @dominikoeh,

thank you for the new Kit and its overall good shape. Next to the two remarks from my side I have a more general question.

Based on the actual contents, I would categorize the maturity of this Kit within the "Incubating Phase". Did you consider to include the contents either within the PURIS or DCM?

**Comment by jSchuetz88** [comment]
@dominikoeh Any updates?

File: docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md (hunk 1 above)
Comment (dominikoeh) [reply docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md:132]: @tom-rm-meyer-ISST can you check this part?

**Comment by dominikoeh** [comment]
> How were the figures generated? Is it possible to provide editable files (ideally puml or mermaid)?

the files used for generation have been added

**Comment by dominikoeh** [comment]
> Hi @dominikoeh,
> 
> thank you for the new Kit and its overall good shape. Based on the actual contents, I would categorize the maturity of this Kit within the "Incubating Phase".
> 
> However, next to the two issues from my side I have a more general question: Did you consider to include the contents either within the PURIS or DCM?

@tom-rm-meyer-ISST can you add some context to this?

File: docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md:132]: Same as in PURIS KIT discussion. We'll keep this as updates on use case related content has been planned for R24.12.

**Comment by tom-rm-meyer-ISST** [comment]
> > Hi @dominikoeh,
> > thank you for the new Kit and its overall good shape. Based on the actual contents, I would categorize the maturity of this Kit within the "Incubating Phase".
> > However, next to the two issues from my side I have a more general question: Did you consider to include the contents either within the PURIS or DCM?
> 
> @tom-rm-meyer-ISST can you add some context to this?

Hi @jSchuetz88 , @dominikoeh: We considered it. The KIT is a common capability of both KITs. Thus, we decided to externalize it and reference it.

From my point of view, the KITs will need further categorization to distinguish e.g. platform capabilities from use cases.

**Review by jSchuetz88 — APPROVED** [review]
Good to go from my side.

@danielmiehle @maximilianong

**Review by danielmiehle — APPROVED** [review]
LGTM - thank you for the contribution!

**Comment by jSchuetz88** [comment]
@tom-rm-meyer-ISST could you merge?

**Pull request merged**